go.mod

```diff
@@ -80,3 +80,5 @@ replace github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-
 
 // TODO review the change introduced by https://github.com/grpc/grpc-go/pull/4416 before upgrading to 1.39.0
 replace google.golang.org/grpc => google.golang.org/grpc v1.38.0
+
+replace github.com/thanos-io/thanos v0.22.0 => github.com/thanos-io/thanos v0.19.1-0.20210923155558-c15594a03c45
```

Comment:
Please, whenever you pin a version, add a comment stating why, and under what conditions we will remove the pin.

Reply:
Pin is used to make sure we use version from master. Otherwise go modules want to use v0.22, which is latest tag, but is not reachable from master, so we would be stuck with version from July.

Reply:
Yeah put it in the file so I can see it and not have to trace back through PRs.

Reply:
https://github.com/cortexproject/cortex/pull/4509